web/directory/management/commands/parse_coop_csv.py

```diff
@@ -177,14 +177,15 @@ def get_city_pks(file_path):
             city = list(city_set)[0]
             zipcode = list(city_set)[1]
             state_id = list(city_set)[2].upper() if list(city_set)[2] != '' else 'IL'
-            print("- model: address.locality")
-            print("  pk:",i)
-            print("  fields:")
-            print("    name: \"",city,"\"", sep='')
-            print("    postal_code: \"",zipcode,"\"", sep='')
-            print("    state: ['", state_id, "', '", country, "']", sep='')
-#            print("    state: 19313")
-            cities_pks[tuple(city_set)] = i 
-            i = i + 1
+            if zipcode != '':
+                print("- model: address.locality")
+                print("  pk:",i)
+                print("  fields:")
+                print("    name: \"",city,"\"", sep='')
+                print("    postal_code: \"",zipcode,"\"", sep='')
+                print("    state: ['", state_id, "', '", country, "']", sep='')
+#                print("    state: 19313")
```

Comment:
Please delete -- I had gotten a little sloppy with commenting things out when I really shoud have been removing the lines.